src/components/common/layout/Separator.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { View } from 'react-native'
+import { withStyles } from '../../../lib/styles'
+
+const Separator = ({ color, width, style, theme }) => (
+  <View
+    style={[
+      {
+        borderBottomColor: theme.colors[color] || color || theme.colors.primary,
+        borderBottomWidth: `${width}px`,
```

Comment:
are we going to use `px` or `normalize`?

Reply:
Done